runtime-config.md

```diff
@@ -1,3 +1,14 @@
+# Runtime Configuration
+
+## Hooks
+
+Lifecycle of a container's runtime, for setup and teardown.
```

Comment:
… and moving over that text gets you the existing “Hooks allow one to run code…”.